### PR TITLE
Removes teratomas

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title. Someone should probably fill in the void this leaves in ling abilites. 😉 

## Why It's Good For The Game

Teratomas are a thing lings can infinitely spawn and the rules allow them to do basically whatever they like in the name of "causing chaos". Being able to just infinitely create antags with no real resources required isn't great, either. I don't think anyone enjoys fighting them or being one.

image

## Changelog
:cl:
del: Removed teratomas
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
could you kindly hold off on this pr until #5624 is resolved? I plan to totally rework tomas into a simplemob instead of a monkey, im just waiting on that pr for icon conflict reasons

**Comment by Wilchenx** [comment]
ided

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -513,18 +513,13 @@ im not even gonna bother with these for the following symptoms. typed em out, co
 				playsound(get_turf(M), 'sound/effects/splat.ogg', 50, 1)
 				if(prob(60))
 					if(tetsuo && prob(15))
-						if(A.affected_mob.job == "Clown")
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/heal.dm:525]: can you not remove the organ spawns from the symptom? read the code you're removing, please?

**Review by zeskorion — CHANGES_REQUESTED** [review]

**Comment by Effemu** [comment]
cope ivan

**Comment by Archanial** [comment]
Issues with teratomas:
- they need absorbs which encourages random murders, and that colides with rules somewhat
- they are spammable

How to solve it:
They don't need to be simplemobs. It's much funnier to be a monkey than a simplemob. Just make it an ability which you can purchase multiple times and each time you do that, you could produce one of the teratomas, which have some sort of mind link to you (or you just understand them) and can possibly be DNA stung to make you somesort of makeshift BB. Of course cost should be adjusted to match their power.

**Comment by zeskorion** [comment]
As the person who added the things in the first place, making them a special simplemob that exists primarily as a nuisance is a better solution. It lets them exist without ghosts, and restricts their capabilities

**Comment by francinum** [comment]
@Archanial The simplemob idea is actually the result of an old discussion on the topic.
No matter what restrictions you think are 'reasonable' on them, the ability for changelings to in any way have the possibility of mass-manufacturing fully fledged carbon antagonists is a poor one.

The simplemob solution solves it and is considered acceptable by the original author.

**Comment by Glcbitj** [comment]
Can't you just make it so there's a big cooldown like 20-30 minutes on spawning them, or make it so they required consumed genomes, like 1/toma. Will make them appear less frequently and probably not do suicidal things anymore.

**Comment by francinum** [comment]
No.

**Comment by ivanmixo** [comment]
uhhhhhhhhhh

**Pull request closed without merging**